components/shared_code/src/shared_data_model/sources/quality_time.py

```diff
@@ -75,6 +75,7 @@
             values=[
                 "Average issue lead time",
                 "Change failure rate",
+                "CI-pipeline duration",
```

Comment:
How do we distinguish between "CI-pipeline" and "CI-jobs" ?

Reply:
According to the user story we're mostly interested in pipeline duration, and only in the case of Jenkins measuring the duration of the jobs is mentioned. And then only as a nice to have. So I think the name is fine for now.